src/CommunityToolkit.Maui.Core/Essentials/SpeechToText/OfflineSpeechToText.shared.cs

```diff
@@ -0,0 +1,29 @@
+using System.Globalization;
+
+namespace CommunityToolkit.Maui.Media;
+
+/// <inheritdoc cref="ISpeechToText"/>
+public static class OfflineSpeechToText
```

Comment:
I wonder if we should add additional `<remarks>` to make it clear to the user that this is providing offline support. I know it's in the name but adding it to intellisense may also help.